app/stac_api/pgtriggers.py

```diff
@@ -0,0 +1,271 @@
+import pgtrigger
+
+AUTO_VARIABLES_FUNC = """
+-- update auto variables
+NEW.etag = gen_random_uuid();
+NEW.updated = now();
+
+RETURN NEW;
+"""
+
+
+def generate_child_triggers(parent_name):
+    '''Function that generates two triggers for parent child relation.
+
+    The triggers update the `updated` and `etag` fields of the parent when a child gets inserted,
+    updated or deleted.
+
+    Returns: tuple
+        Tuple of Trigger
+    '''
+    child_update_func = """
+    -- update related {parent_name}
+    UPDATE stac_api_{parent_name} SET
```

Comment:
fine if we will nave have to update a table different from that schema ``stac_api_<table>``

Reply:
@ltclm Currently in service-stac all tables are prefixed with `stac_api_`, this prefix is the name of the django application that we gave. So instead if we rename the django application and/or add a second django application, this prefix won't change.